src/libpanic_abort/lib.rs

```diff
@@ -8,6 +8,7 @@
 #![doc(html_root_url = "https://doc.rust-lang.org/nightly/",
        issue_tracker_base_url = "https://github.com/rust-lang/rust/issues/")]
 #![panic_runtime]
+#![cfg_attr(not(bootstrap), rustc_panic_abort_runtime)]
```

Comment:
Won't this in turn break cargo's `-Zbuild-std` which builds this crate with the unwind strategy?

Reply:
Building this crate with the `unwind` strategy is the wrong behavior - it should always be built with the `abort` strategy. So, this will correctly change how `-Zbuild-std` builds this crate - but it's a bugfix, not breakage.